### PR TITLE
Update Multiplication.cs

### PR DESCRIPTION
**Pull request by ramahaw**

### CODE REVIEW

File: Multiplication.cs [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
 namespace multiplication
 {
     class Multiplication
     {
         static void Main(string[] args)
         {
-            // This program will give you the multiplication table of any number you enter
-
-            Console.WriteLine("Please enter a number");
+            Console.WriteLine("Please enter the number you want to know the multiplication table for him");
```
Comment (ab-dulla) [inline Multiplication.cs:8]: Him? It is not a person.

`Please enter a number to print its multiplication table`

File: Multiplication.cs [hunk 2]
```diff
@@ -1,18 +1,11 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
```
Comment (ab-dulla) [inline Multiplication.cs:6]: Have you already tested that it is still woking?

**Review by ab-dulla — CHANGES_REQUESTED** [review]

File: Multiplication.cs (hunk 2 above)
Comment (ramahaw) [reply Multiplication.cs]: Yes, I did

**Comment by ab-dulla** [comment]
This one is duplicated https://github.com/ramahaw/c-sharp-exercises/pull/14

**Pull request closed without merging**